cmd/ana/main.go

```
@@ -95,7 +101,7 @@ func run(args []string, stdio cli.IO, env func(string) string) error {
 		// by wrapping cli.ErrUsage.
 		if errors.Is(rerr, config.ErrUnknownProfile) {
 			fmt.Fprintf(stdio.Stderr, "ana: unknown profile %q\n", global.Profile)
-			return fmt.Errorf("%w: %s", cli.ErrUsage, rerr)
+			return errors.Join(fmt.Errorf("%w: %s", cli.ErrUsage, rerr), cli.ErrReported)
```

golangci-lint (2.11.4)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)